lib/core/lib/ops/iam.js

```diff
@@ -42,8 +42,48 @@ module.exports = function (policies) {
     }
   }
 
+  function actionsOnResources ({ resources }, done) {
+    try {
+      const resultMap = {}
+
+      const statements = _(policies).reduce((statements, policy) => {
+        return _.concat(statements, policy.Statement)
+      }, [])
+
+      resources.forEach((resource) => {
+        resultMap[resource] = []
+        statements.forEach((statement) => {
+          if (statement.Effect === 'Allow') {
+            statement.Resource.forEach((r) => {
+              if (pbac.conditions.StringLike(r, resource)) {
+                statement.Action.forEach((action) => {
+                  isAuthorized({resource, action}, (err, access) => {
+                    if (err) return
```

Comment:
Should this error be swallowed?

Reply:
@dberesford Yes I wasn't too happy about that piece there. However if you look at the isAuthorized function which is declared in the same file:

```
  function isAuthorized (params, done) {
    return done(null, pbac.evaluate(params))
  }
```

You can see that the `err` can never actually come back. I guess I had that return statement in there to avoid linting issues. What do you think?

Reply:
ah I see, I would refactor that method, probably get rid of it altogether. 

Thanks for making the doc changes!